rel/files/eunit.config

```diff
@@ -13,5 +13,18 @@
 [
     {kernel, [{error_logger, silent}]},
     {sasl, [{sasl_error_logger, false}]},
-    {fabric, [{eunit_run, true}]}
+    {fabric, [{eunit_run, true}]},
```

Comment:
Looks like the trailing comma here is causing your parse error. Maybe move your comment and config value before the eunit_run setting to avoid that?

Reply:
Oh sheesh that's a dumb mistake on my last fixup of this commit adding the extra explanation for the `test_cluster_file`. Will fix.